test/e2e/instrumentation/monitoring/custom_metrics-deployments.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Why is it `custom_metrics-deployments.go` and not `custom_metrics_deployments.go`?

Reply:
Same comment to the other file

Reply:
Renamed, this is by mistake.